config/settings/base.py

```diff
@@ -47,24 +48,20 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    },
-    'production': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': os.environ.get('POSTGRES_DB'),
-        'USER': os.environ.get('POSTGRES_USER'),
-        'PASSWORD': os.environ.get('POSTGRES_PASSWORD'),
-        'HOST': os.environ.get('POSTGRES_HOST', 'localhost'),
-        'PORT': os.environ.get('POSTGRES_PORT', '5432'),
-        'SSL_MODE': os.environ.get('POSTGRES_SSLMODE', 'disable'),
-        'TEST': {
-            'NAME': 'song_lyrics_test'
-        }
-    },
```

Comment:
bro if you remove this , what will we use when we deploy the application into production...

Reply:
I replaced it with `dj_database_url.config()` so as to have a consistent setting across the development and production environment.